charts/longhorn-crd/100.3.1+up1.4.1/templates/validate-psp-install.yaml

```diff
@@ -0,0 +1,7 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
```

Comment:
We don't need the PSP validate logic in this chart as it will be validated in the main `longhorn` chart